src/components/Resume/ResumeDetail/ResumeDetail.jsx

```diff
@@ -0,0 +1,124 @@
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import TextField from "@material-ui/core/TextField";
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import React, { useState } from "react";
+import { GetUser } from "./ResumeApi";
+import "./ResumeDetail.css";
+import { useEffect } from "react";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    width: 500,
+  },
+  margin: {
+    "& > *": {
+      margin: theme.spacing(1),
+    },
+  },
+}));
+
+function Resume({ userId }) {
+  const [user, setUser] = useState();
+
+  const refresh = () => {
```

Comment:
refresh function is resolving a promise coming from your api call. You need to handle the error and possible results coming from your ResumeApi -> GetUser()